package.json

```
@@ -11,7 +11,8 @@
     "start": "start-storybook -p 6006",
     "build": "tsc",
     "preversion": "npm run build",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "type-check": "node_modules/.bin/tsc --noEmit"
```

for consistency with other scripts, we can omit the `node_modules/.bin` prefix, and let the package manager resolve it for us!

```suggestion
    "type-check": "tsc --noEmit"
```

lmk If i missed something!!

-------

btw, talking about package managers, this repo used to use `yarn` then we switched to `npm` and maybe we switched back to `yarn` - i forgot tbh :/ so you'll notice we have npm scripts in the package.json, like `"preversion": "npm run build"`, at the same time we have `yarn.lock` instead of `package-lock.json`, and we have `resolutions` in package.json, which only works with yarn (npm uses `overrides`) - so you get it 😄 we're mixing yarn and npm! so we'll need to fix that inconsistency; if you can do some research and choose one of them! (ps: pnpm is another option, tried it before, but it caused some problems when using this package in the main web app)

---

got it ok thanks for the context ill make sure to look into npm and yarn

---

awesome thanks! please note that yarn has 2 versions, [classic (v1)](https://github.com/yarnpkg/yarn) and a [new version (v2)](https://github.com/yarnpkg/berry) - we use v1 in the main app, as v2 is fundamentally different from yarn v1 and npm, and it's not too mature yet i think